lstchain/scripts/lstchain_mc_dl1_to_dl2.py

```diff
@@ -62,6 +62,15 @@ def main():
     config = replace_config(standard_config, custom_config)
 
     data = pd.read_hdf(args.datafile, key=dl1_params_lstcam_key)
+
+    # Dealing with pointing errors
```

Comment:
To try to keep track of these problems, I would be more specific here, these are not really errors generated by a lack of pointing information, but due to missing time stamp

Reply:
Added a comment.